src/style.css

```diff
@@ -0,0 +1,88 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  box-sizing: border-box;
+  height: 60px;
+  position: relative;
+  margin: 0;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav_list {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  margin: 0;
+  list-style-type: none;
+}
+
+.nav_list:last-child {
+  margin-right: 50px;
+}
+
+.nav_item {
+  line-height: 60px;
+  margin-left: 20px;
+}
+
+.link:hover,
+.link:active {
+  color: #00acdc;
+}
+
+.link:active::after {
+  content: "";
+  position: absolute;
+  display: block;
+  width: 100%;
+  border-bottom: 4px solid #00acdc;
+  border-radius: 4px;
+  bottom: 0;
+}
+
+.is-active {
+  color: #00acdc !important;
```

Comment:
NO! don't use important!

Reply:
I use !important only for tests 'class="is-active"'

Reply:
why you need `important` for this test?

Reply:
rewrite style without important

Reply:
Because, I need more specificity for this class.
I rewrite this style.